ecdsa/resharing/round_1_old_step_1.go

```diff
@@ -94,9 +94,6 @@ func (round *round1) Update() (bool, *tss.Error) {
 	}
 	// accept messages from old -> new committee
 	for j, msg := range round.temp.dgRound1Messages {
-		if round.oldOK[j] {
```

Comment:
This check prevents players from the old committee to pass ECDSA public key to themselves in the new committee at the end of this block. 

This is a bit counterintuitive but is needed since `round.input` (recovered from file) and `round.save` (used in later rounds) play different roles for old players here. Hence, the transition of ECDSAPub from `input` to `save` must happen here.

For _new-new_ players this happens via broadcast over p2p transport, and for _old-new_ players same must occur via go channels abstraction.